_tools/packaging/prepare-rpm-build.sh

```diff
@@ -6,11 +6,7 @@ pwd=`dirname $0`
 . "$pwd/common.sh"
 
 MACKEREL_AGENT_NAME=${MACKEREL_AGENT_NAME:-mackerel-agent}
-spec_filename="mackerel-agent.spec"
-if [ "$BUILD_SYSTEMD" != "" ]; then
-    spec_filename="mackerel-agent-systemd.spec"
-fi
```

Comment:
Is better to delete `BUILD_SYSTEMD=1` environment variables?

https://github.com/mackerelio/mackerel-agent/blob/23d227960f622922178a2f96de57643998524743/Makefile#L126-L135

Reply:
Thanks, sure. 
I fixed at 6171c1528a1551b664e7b4c1c956e086c25527ab 79e7d6c651bf97b89db834523c51d959c16f3431